### PR TITLE
Removed '*Options' suffixes from 'Abstractions.Cluster' properties

### LINKED ISSUE #396: Remove '*Options' suffixes from 'Abstractions.Cluster' properties
There are inconsistently used '*Options' suffixes on [Cluster properties](https://github.com/microsoft/reverse-proxy/blob/master/src/ReverseProxy/Abstractions/ClusterDiscovery/Contract/Cluster.cs#L26). It's suggested to remove them.

### PR DESCRIPTION
**Pull request by Marusyk**
Removed '*Options' suffixes 

Fixes #396

Please review
Thank you in advance

### CODE REVIEW

File: src/ReverseProxy/Abstractions/ClusterDiscovery/Contract/Cluster.cs [hunk 1]
```diff
@@ -23,17 +23,17 @@ public sealed class Cluster : IDeepCloneable<Cluster>
         /// <summary>
         /// Circuit breaker options.
         /// </summary>
-        public CircuitBreakerOptions CircuitBreakerOptions { get; set; }
+        public CircuitBreakerOptions CircuitBreaker { get; set; }
```
Comment (Tratcher) [inline src/ReverseProxy/Abstractions/ClusterDiscovery/Contract/Cluster.cs:26]: Do some of these sound odd in the singular?
```suggestion
        public CircuitBreakerOptions CircuitBreakers { get; set; }
```
Also HealthChecks, Quotas. The others seem ok.

**Comment by Kahbazi** [comment]
I think the docs also needs to be changed.

https://github.com/microsoft/reverse-proxy/blob/87e463ca4e64aa1c1b48c13881a4e41bb334d954/docs/docfx/articles/proxyhttpclientconfig.md#L14

File: src/ReverseProxy/Abstractions/ClusterDiscovery/Contract/Cluster.cs (hunk 1 above)
Comment (Marusyk) [reply src/ReverseProxy/Abstractions/ClusterDiscovery/Contract/Cluster.cs:26]: So, should I change it to CircuitBreakers, HealthChecks, and Quotas?

Comment (Tratcher) [reply src/ReverseProxy/Abstractions/ClusterDiscovery/Contract/Cluster.cs:26]: Yes, unless you can give a reason not to?

Comment (Marusyk) [reply src/ReverseProxy/Abstractions/ClusterDiscovery/Contract/Cluster.cs:26]: I would prefer to leave it as is.

CircuitBreaker - [Circuit breaker design pattern](https://en.wikipedia.org/wiki/Circuit_breaker_design_pattern), but not Circuit breakers. 
HealthCheck - we set up options of one health check (timeout, interval etc.). There is no possibility to set up many checks, so, HealthCheck seems to be ok.
Quotas - don't know, can be changed

Comment (Tratcher) [reply src/ReverseProxy/Abstractions/ClusterDiscovery/Contract/Cluster.cs:26]: Fair enough. We'll do an end-to-end API review later in the release.

**Review by Tratcher — APPROVED** [review]

**Comment by Tratcher** [comment]
Thanks

**Pull request merged**